gui/wxpython/core/utils.py

```diff
@@ -827,7 +827,7 @@ def StoreEnvVariable(key, value=None, envFile=None):
     if os.path.exists(envFile):
         try:
             fd = open(envFile)
-        except OSError as e:
+        except OSError:
             sys.stderr.write(_("Unable to open file '%s'\n") % envFile)
```

Comment:
I think it would be better to actually report the error:

```suggestion
        except OSError as error:
            sys.stderr.write(_("Unable to open file '{name}': {error}\n").format(name=envFile, error=error)
```

Reply:
This and the following have been updated.